runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6659,8 +6657,25 @@ TR_J9ByteCodeIlGenerator::genAconst_init(TR_OpaqueClassBlock *valueTypeClass, in
       // If the class is still unresolved, abort the compilation and track the failure with a static debug counter.
       abortForUnresolvedValueTypeOp("aconst_init", "class");
       }
+   else if (comp()->fej9()->isClassInitialized(valueTypeClass))
```

Comment:
It might be useful to add an environment variable that would prevent use of the static default value instance, just in case we want to experiment further with its effect on performance.

Reply:
Addressed in 013aa3e